src/proxy.ts

```diff
@@ -2,6 +2,25 @@ import * as debug from 'debug';
 
 const d = debug('@electron/get:proxy');
 
+/**
+ * get process.env variable
+ * ignore case and compatible `GLOBAL_AGENT_` namespace
+ */
+function getProxyEnv(name: string): string | undefined {
+  const namespace = 'GLOBAL_AGENT_';
+  const namespaceProxyName = `${namespace}${name}`;
+
+  for (const envName in process.env) {
+    if (namespaceProxyName.toLowerCase() === envName.toLowerCase()) {
+      return process.env[envName];
+    }
+
+    if (name.toLowerCase() === envName.toLowerCase()) {
+      return process.env[envName];
+    }
+  }
```

Comment:
Seems like this can be simplified:
```suggestion
  const possibleKeys = [namespaceProxyName.toLowerCase(), name.toLowerCase()];
  const matchingKey = Object.keys(process.env).find((key) => possibleKeys.includes(key.toLowerCase()));

  if (matchingKey) {
    return process.env[matchingKey];
  }
```

Reply:
This approach would indeed be simplified, but it implicitly has three loops: `keys`, `find`, `includes`.
At the same time, I found a problem with this method: it does not respect the policy that GLOBAL_AGENT_ has a higher priority.

Let me change that